include/glow/Partitioner/Partitioner.h

```diff
@@ -296,6 +296,9 @@ class Partitioner {
 
   /// Get function for computeTime_
   ComputeTimeMapTy getComputeTime() const { return computeTime_; }
+
+  /// Get function for memUsage_
+  MemUsageMapTy getMemUsage() const { return memUsage_; }
```

Comment:
any immediate usage?

Reply:
We want to use this in a partitioner optimization for load balance. That PR is on its way.